frontend/pages/ManageControlsPage/Scripts/components/ScriptListItem/ScriptListItem.tsx

```diff
@@ -80,15 +80,16 @@ const ScriptListItem = ({
 
   const { graphicName, platform } = getFileRenderDetails(script.name);
 
+  const onClickEdit = (evt: React.MouseEvent | React.KeyboardEvent) => {
+    evt.stopPropagation();
```

Comment:
Ensures that clicking the edit icon opens the edit script modal instead of script details modal, while preserving the latter functionality when clicking anywhere else in the script list item.

Reply:
Same for other 2 action handlers